### DISCUSSION TITLE
Why I got message on banner? "Please configure another 2FA method to reduce your risk of permanent account lockout. We strongly recommend against SMS as it is prone to fraud and delivery may be unreliable depending on your region."

### DISCUSSION THREAD

**Discussion by baweiji** [community/community#129152] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have setup Authenticator app under Two-factor methods. Why I still get this message?

**Comment by Jorgeluisreis** [comment]
I also received it, the problem is that my country doesn't support SMS (Brazil) and I already use 2FA

**Comment by ouuan** [comment, marked as answer]
See #129244. It's saying that you need more than one 2FA method. The warning will go away if you add one more 2FA method. However, I don't want to do that.

**Comment by ghoshvidip26** [comment]
Same here. I also got the same message

**Comment by Hossein-Ak** [comment]
Why should we add another 2FA? and what are the redlines for perma lockout?

**Reply by pkpro** [reply]
That is what I also want to know.

**Comment by pkpro** [comment]
I suppose MS is pushing for "GitHub Mobile app", because they "worried sick" that google also getting your logging behavior if you are using google Authenticator ...

**Comment by golddydev** [comment]
I got the same message

**Comment by VuTheQuan** [comment]
I got the same message!

**Comment by golddydev** [comment]
I added Git Mobile, and that warning is disappeared.

**Comment by hhhawa** [comment]
I got the same

**Comment by Mercerenies** [comment]
I'm confused by this message, because three out of the four authentication options involve my mobile device. There's no sense in having a TOTP app _and_ GitHub Mobile set up, because if I lose my mobile phone then I'm out of luck on both of those. The redundancy here doesn't actually gain me anything, unless I happen to have a webauthn-compatible device lying around.

**Comment by AAdewunmi** [comment]
Got the same message. Added GitHub mobile and message disappeared!
